web/wp-content/themes/dctx-v2/inc/acf.php

```diff
@@ -0,0 +1,449 @@
+
+	// Only try to get the rest of the settings if the background type is set to anything.
+	if ( $args['background_type'] ) {
+		if ( 'color' === $args['background_type'] && $background_options['background_color']['color_picker'] ) {
+			$background_color = $background_options['background_color']['color_picker'];
+			$args['class']   .= ' has-background color-as-background background-' . esc_attr( $background_color );
+		}
+
+		if ( 'image' === $args['background_type'] && $background_options['background_image'] ) {
+			$background_image = $background_options['background_image'];
+
+			// Construct class.
+			$args['class'] .= ' has-background image-as-background' . esc_attr( $has_show_overlay );
+			ob_start();
+			?>
+			<figure class="image-background" aria-hidden="true">
+				<?php echo wp_get_attachment_image( $background_image['id'], 'full' ); ?>
+			</figure>
+			<?php
+			$background_image_markup = ob_get_clean();
+		}
+
+		if ( 'video' === $args['background_type'] ) {
+			$background_video      = $background_options['background_video'];
+			$background_video_webm = $background_options['background_video_webm'];
+			$background_title      = $background_options['background_video_title'];
+			$args['class']        .= ' has-background video-as-background' . esc_attr( $has_show_overlay );
+			// Translators: get the title of the video.
+			$background_alt = $background_title ? sprintf( esc_attr( 'Video Background of %s', 'dctx' ), esc_attr( $background_options['background_video_title'] ) ) : __( 'Video Background', 'dctx' );
+
+			ob_start();
+			?>
+				<video class="video-background" autoplay muted loop preload="auto" aria-hidden="true"<?php echo $background_title ? ' title="' . esc_attr( $background_title ) . '"' : ''; ?>>
+						<?php if ( $background_video_webm['url'] ) : ?>
+						<source src="<?php echo esc_url( $background_video_webm['url'] ); ?>" type="video/webm">
+						<?php endif; ?>
+
+						<?php if ( $background_video['url'] ) : ?>
+						<source src="<?php echo esc_url( $background_video['url'] ); ?>" type="video/mp4">
+						<?php endif; ?>
+				</video>
+				<button class="video-toggle"><span class="screen-reader-text"><?php esc_html_e( 'Toggle video playback', 'dctx' ); ?></span></button>
+			<?php
+			$background_video_markup = ob_get_clean();
+		}
+
+		if ( 'none' === $args['background_type'] ) {
+			$args['class'] .= ' no-background';
+		}
+	}
+
+	// Set the custom font color.
+	if ( $args['font_color']['color_picker'] ) {
+		$args['class'] .= ' has-font-color color-' . esc_attr( $args['font_color']['color_picker'] );
+	}
+
+	// Print our block container with options.
+	printf(
+		'<%s id="%s" class="%s">',
+		esc_attr( $args['container'] ),
+		esc_attr( $args['id'] ),
+		esc_attr( $args['class'] )
+	);
+
+	// If we have a background video, echo our background video markup inside the block container.
+	if ( $background_video_markup ) {
+		echo $background_video_markup; // WPCS XSS OK.
```

Comment:
:warning: **Warning**: Using the WPCS native whitelist comments is deprecated. Please use the PHPCS native &quot;phpcs:ignore Standard.Category.SniffName.ErrorCode&quot; annotations instead. Found: // WPCS XSS OK.\n (*WordPress.Security.EscapeOutput.DeprecatedWhitelistCommentFound*).